spec/models/topic_spec.rb

```diff
@@ -0,0 +1,51 @@
+# == Schema Information
+#
+# Table name: topics
+#
+#  id          :bigint           not null, primary key
+#  description :string(50)       not null
+#  image       :string
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#
+require 'rails_helper'
+
+RSpec.describe Topic, type: :model do
+	let!(:topics) do
+		[Topic.create(:description => "Music", :image => "image string 1"), Topic.create(:description => "Sports", :image => "image string 2"), Topic.create(:description => "Food", :image => "image string 3")]
+	end
+
+	let(:topic) do
+		Topic.create(:description => "Plants", :image => "image string here")
+	end
+
+	# TODO: See if repetition of 'find_by to/not_to be_nil' expectations should be avoided
+
+  it { should validate_presence_of(:description) }
+
+  it "is expected to save topics successfully after #create" do
+		expect(Topic.all).to match_array(topics)
+		expect(Topic.count).to eq(3)
+	end
+
+	it "is expected to save description successfully after #update" do
+		topic.description = "Zombies"
+		topic.save
+		expect(Topic.find_by(:description => "Plants")).to be_nil
```

Comment:
try using the new syntax, that is `Topic.find_by(description: 'Plants')`
For example here you are testing if `saves` works correctly. That is something that is already tested on ActiveRecord class. You inherited from ActiveRecord when creating your model class and you can assume `save` mehtod will work fine.